language-plutus-core/src/Language/PlutusCore/Evaluation/Machine/ExBudgeting.hs

```diff
@@ -149,9 +149,7 @@ class ToExMemory term => SpendBudget m term | m -> term where
     -- 1. do nothing for an evaluator that does not care about costing
     -- 2. count upwards to get the cost of a computation
     -- 3. subtract from the current budget and fail if the budget goes below zero
-    --
-    -- The @term@ argument is only used for reporting an error.
-    spendBudget :: ExBudgetCategory -> term -> ExBudget -> m ()
+    spendBudget :: ExBudgetCategory -> ExBudget -> m ()
```

Comment:
I remove the `term` argument just to make things type check. I discussed this with @reactormonk and I don't think there's a terrible need for that argument right now, so I think it's fine to keep it removed, but if it's not, it should be possible to restore it.